samples/aspnetcore-integration-tests/AspNetCoreSample.Tests/ReadEnvironmentFixture.cs

```diff
@@ -0,0 +1,27 @@
+﻿using MongoDB.Driver;
+using System;
+using System.Threading.Tasks;
+using TestEnvironment.Docker.Containers.Mongo;
+
+namespace AspNetCoreSample.Tests
+{
+    public class ReadEnvironmentFixture : EnvironmentFixture
```

Comment:
Maybe it worth to rename them to ReadOnly*/WriteOnly* to make names more clear?

Reply:
Well, `Write` fixture also allows to read, so `Only` suffix sounds too strict.